src/data/data.js

```diff
@@ -266,6 +266,45 @@ const pageData = {
       },
     ],
   },
+  registro: {
+    sections: [
+      {
+        component: 'registerFormComp',
+        headline: 'Regístrate',
+        subheadline: 'Nos pondremos<br class="form__break"> en contacto contigo.',
```

Comment:
Do not add markup in the data. If you need to alter the presentation of the info, please use CSS to do so.

Reply:
I tried that way but the flow of the text doesn't break like the one presented in the designs if I set a max-width for it.